doc/internals.rst

```diff
@@ -133,6 +133,8 @@ They may be useful points of reference for your work:
 - `windspharm <http://ajdawson.github.io/windspharm/index.html>`_: Spherical
   harmonic wind analysis in Python.
 - `eofs <http://ajdawson.github.io/eofs/>`_: EOF analysis in Python.
+- `salem <http://salem.readthedocs.io>`_: extends xarray to add geolocalised
```

Comment:
Sure. What do you have in mind for the more user facing page? And which of the packages are ready for prime-time? Salem is surely the less ready here, but there are also others listed here: https://aospy.hackpad.com/Summary-of-1st-brainstorming-round-lLJ3vXhCk7Z#:h=Potentially-related-or-relevan